modules/microshift-install-rhel-image-mode-conc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-rhel-image-mode-conc_{context}"]
+= Image mode for {op-system-base-full}
+
+Image mode for {op-system-base-full} is a technology preview deployment method that uses a container-native approach to build, deploy and manage the operating system as a bootc image. With bootc, you can build, deploy, and manage the operating system as if it is any other container.
+
+* This container image uses standard OCI/Docker containers as a transport and delivery format for base operating system updates.
+* A bootc image includes a Linux kernel, which is used to start the operating system.
+* By using bootc containers, developers, operations administrators, and solution providers can all use the same container-native tools and techniques.
+
+Image mode splits the creation and installation of software changes into two steps: one on a build system and one on a running target system.
+
+* In the build-system step, a Podman build inspects the RPM files available for installation, determines any dependencies, and creates an ordered list of chained steps to complete, with the end result being a new operating system available to install.
+
+* In the running-target-system step, a bootc update downloads, unpacks, and makes the new operating system bootable alongside the currently running system. Local configuration changes are carried forward to the new operating system, but do not take effect until the system is rebooted and the new operating system image replaces the running image.
+
+[id="microshift-install-rhel-image-mode-conc_{context}"]
+== Using image mode for {op-system} with {microshift-short}
+
+To use image mode for {op-system}, ensure that the following resources are available:
+
+* A {op-system} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images.
+* A remote registry for storing and accessing bootc images.
+
+The workflow for using image mode with {microshift-short} includes the following steps:
+
+. Build the {microshift-short} bootc image.
+. Publish the image.
+. Run the image. This step includes configuring {microshift-short} networking and storage.
+
+[IMPORTANT]
+====
+The `rpm-ostree` file system is not supported in image mode and must not be used to make changes to deployments that use image mode.
```

Comment:
Actually, bootc uses `rpm-ostree` file system for managing its layers. Do I understand correctly that we want to say that we should not use `rpm-ostree` command, not file system?

Reply:
It's from the [RHEL docs](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/using_image_mode_for_rhel_to_build_deploy_and_manage_operating_systems/index#introducing-image-mode-for-rhel_using-image-mode-for-rhel-to-build-deploy-and-manage-operating-systems).

"Warning: The use of rpm-ostree to make changes, or install content, is not supported."

I assumed this means that you can't mix RHEL for Edge with image mode. The two are not synonymous. What should we tell users about the difference between the two, do you think?

Reply:
Should we mention lack of rollback, for example?

Reply:
No, the warning means that the `rpm-ostree` command should not be used for managing bootc layers.
I do not think we should go into the differences here - let's just add the warning.